src/firstboot/firstboot.c

```diff
@@ -837,6 +839,17 @@ int main(int argc, char *argv[]) {
 
         umask(0022);
 
+        r = get_proc_cmdline_key("systemd.firstboot=", &enabled);
```

Comment:
why that? so far the primary place where we documented kernel command line options was kernel-command-line(7), and I figure this really belongs there too, but not in systemd(1) which only covers the options relevant to PID 1 itself.